strax/chunk.py

```diff
@@ -332,3 +360,29 @@ def split_array(data, t, allow_early_split=False):
         t = min(data[splittable_i]['time'], t)
 
     return data[:splittable_i], data[splittable_i:], t
+
+
+@export
+def transform_chunk_to_superrun_chunk(superrun_id, chunk, prev_chunk_end=None):
```

Comment:
Daniel, would you mind explaining why we need this? I'm a bit hesitant to say it's correct as it appears you could have the following situation:

```
run 1: |-----|----------
run 2: -------|----|----
run 3: -------------|--|
r1+3 : |-----|---------|
```
where the start time of the subrun3 chunk is well before the start time of that subrun. Wouldn't this be a bit strange and possibly cause funny things later?

Reply:
In case if both chunks from run 1 and run 3 are 200 MBs you are right. But I disagree with you. I think it would be better to adjust the chunk boundaries for a superrun chunk to be consistent with the data handling  for regular runs. 

> Wouldn't this be a bit strange and possibly cause funny things later?

What kind of issues do you suspect, maybe I forgot something? Actually, I thought it is saver in this way I implemented it, as I am not sure what happens if you query time_ranges outside of any chunk boundary. 

From a physics point of view it should not matter if there are any gaps in your data. You just have to compute your livetime correctly and therefore I added the subrun field to our chunks as well as the livetime field to the superrun run-metadata.

Reply:
> I might now see why you made this transform_chunk_to_superrun_chunk function (see above). But would the issue you are trying to solve not much easier be addressed in continuity_check?

I also thought about that, but see my comment above.

Reply:
Chunking issues can be nasty. E.g. if you were to combine super-runs into super-runs by adding `run2` to `r1+3`, you would have overlapping chunks.

>I think it would be better to adjust the chunk boundaries for a superrun chunk to be consistent with the data handling for regular runs.

But this feels very patchy right? I think just adding a line in the continuity check to see if the subrun is different from one chunk to another would be more robust. Also you wouldn't then have to worry about how you compute the live-time, you would get it correctly either from looking at the chunks or the docs. Also I think it's very strange to have chunks of a subrun extending well beyond the boundaries of that run and can for sure lead to confusion/errors later.

Is there any compelling argument why a superrun should have continuous chunks even tough we know that for a fact, there is no reason to assume the chunks are continuous from run to run. If it is only the continuity check which motivates this?

>In case if both chunks from run 1 and run 3 are 200 MBs you are right.

Sorry, I don't follow this point?